api/elastic.py

```diff
@@ -343,13 +343,23 @@ def createFilterClausesAndAggregations(self, filterParams):
         languageFilters = list(filter(
             lambda x: x[0] == 'language', filterParams)
         )
-        formatFilters = list(filter(lambda x: x[0] == 'format', filterParams))
+        formatFilters = list(filter(
+            lambda x: x[0] == 'format', filterParams)
+        )
         displayFilters = list(filter(
             lambda x: x[0] == 'showAll', filterParams)
         )
+        govDocFilters = list(filter(
+            lambda x: x[0] == 'govDoc', filterParams)
+        )
+        noGovDocFilters = list(filter(
+            lambda x: x[0] == 'noGovDoc', filterParams)
+        )
```

Comment:
I would make this a single `govDoc` filter and have that have 3 valid values:
1. `onlyGov` returns only government documents
2. `nonGov` Returns only NON government documents
3. `all` Returns everything regardless (the default if this filter isn't set, but still need an explicit value IMO)